wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/AddParticipantActionHandler.swift

```diff
@@ -153,6 +153,7 @@ class AddParticipantActionHandler: ActionHandler<AddParticipantAction> {
             }
             Task {
                 await eventProcessor.processConversationEvents([updateEvent])
+                await context.perform { self.context.enqueueDelayedSave() }
```

Comment:
What is the benefit of this func over standard `try context.save` or our other convenience `context.saveOrRollback`?

Reply:
`context.enqueueDelayedSave()` is a throttled/debouncing save which makes it safe to use in a loop without any performance penalty, for example when we are processing list of events.  

https://github.com/wireapp/wire-ios/blob/cfb8f53e307b3f15ebd88ff9caa9dda6273f1ed3/wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext%2Bzmessaging.m#L413

Now, I don't think `AddParticipantActionHandler` participates in the event processing so I could actually use `saveOrRollback`here.

Reply:
Changed it to `saveOrRollback()`